src/components/AssasinsSelection.js

```diff
@@ -47,9 +46,8 @@ const AssasinsSelection = (props) => {
             const index = targetArray.indexOf(killedPlayerNamed);
             const spliced = targetArray.slice();
             spliced.splice(index, 1);
```

Comment:
Is spliced getting used anymore?

Reply:
Yeah, it wasn't. I think there should be no more extra lines.